docs/azure-prices.md

```diff
@@ -0,0 +1,123 @@
+---
+sidebar_position: 8
+---
+Azure Price Configuration
+============
+
+OpenCost needs access to the Microsoft Azure Billing Rate Card API to access accurate pricing data for your Kubernetes resources.
```

Comment:
May just drop a note that this is "accurate at time of data access" since discounts can be applied after the fact.